modules/fedimint-ln-client/src/receive.rs

```diff
@@ -88,7 +88,7 @@ impl State for LightningReceiveStateMachine {
 pub struct LightningReceiveSubmittedOffer {
     pub offer_txid: TransactionId,
     pub invoice: Bolt11Invoice,
-    pub payment_keypair: KeyPair,
+    pub receiving_key: ReceivingKey,
```

Comment:
This isn't backwards compatible, so either relies on state machine migrations #3417 or needs to be implemented as a new, separate state machine.